api/controllers/index.js

```diff
@@ -2,12 +2,12 @@ const express = require('express');
 const router = express.Router();
 
 // Load each controller
-const postsController = require('./posts.js');
+const postsController = require('./items.js');
```

Comment:
Just to make it consistent, I think you should change "postsController" to "itemsController"

Reply:
Definitely, fixed it.